scrapd/core/apd.py

```diff
@@ -533,6 +532,9 @@ def parse_page_content(detail_page, notes_parsed=False):
     if date_field_str:
         d[Fields.DATE] = date_utils.parse_date(date_field_str)
 
+    # Parse the `Deceased` field.
+    d[Fields.DECEASED] = parse_deceased_field(normalized_detail_page)
```

Comment:
Apply the same pattern as for the date field (l531-532). You can also look at my other PR which does something similar: https://github.com/scrapd/scrapd/pull/138/files#diff-2de3abbf558189989f23bce31a4003f1R528.